policies/README.md

```diff
@@ -0,0 +1,64 @@
+# OPA policies
+Policies are rules that can be applied when any of your cluster object is _createdn_, _updated_, _patched_ and _deleted_.
+Policies are always executed _before_ the action takes place and their goal is to allow or prevent the requested modification.
+
+This solution is very fast in term of required computing time thanks to the efficienty of the OPA gatekeeper.
+
+
+## How to use policies
+To use OPA gatekeeper you need to:
+- Deploy a Release using Prebuilt Image. See [link](../infrastructure/policy-enforcement/README.md).
+- Create your own constraint, template and eventually config-sync. See [examples](../infrastructure/policy-enforcement/examples).
+
+Now you can test your policies on your cluster objects.
+
+
+## How to use the Gatekeeper
+
+### Template
+In the Template you should define your own constraint CRD and associate it to your rego policies, such as in this [**template.yaml**](../infrastructure/policy-enforcement/examples/template.yaml) example.
+
+### Constraint
+In Constraint you should declare on which resources your Template must be enforced and define your parameters values, such as in this [**constraint.yaml**](../infrastructure/policy-enforcement/examples/constraint.yaml) example.
+You can retrieve your input object using `input` variable.
+
+### Config-Sync
+If you need to get resources from the cluster to write your policy, you need first to cache those and then you will be able to retrieve them in the Template rules using `data.inventory`, such as in this [**config-sync.yaml**](../infrastructure/policy-enforcement/examples/config-sync.yaml) example.
+
+
+### Retrieve cached resources
+To retrieve the cached resources in the rego rules you can do as follow:
```

Comment:
Provide some more context. For a newby reader, those variables mean almost nothing and he has no idea what to do with them.